pkg/util/common.go

```diff
@@ -104,7 +108,7 @@ func FakeGOPATH(srcDir string) (string, error) {
 	log.Printf("GOPATH: %s", os.Getenv("GOPATH"))
 
 	gitRoot := fmt.Sprintf("%s/src/k8s.io", baseDir)
-	if err := os.MkdirAll(gitRoot, 0o755); err != nil {
+	if err := os.MkdirAll(gitRoot, os.FileMode(int(0755))); err != nil {
```

Comment:
Hm, what is the benefit of doing it like this? :upside_down_face:

Reply:
I was getting a compiler error on my local, but I think my config was just wrong. I'll change back :)

Reply:
Okay, maybe you have no go 1.13 on your local machine?